runtime/zeitgeist/src/parachain_params.rs

```diff
@@ -63,14 +46,13 @@ parameter_types! {
     pub const SignatureNetworkIdentifier:  &'static [u8] = b"zeitgeist-";
 
     // Cumulus and Polkadot
-    pub Ancestry: MultiLocation = Junction::Parachain(ParachainInfo::parachain_id().into()).into();
+    pub Ancestry: MultiLocation = Parachain(ParachainInfo::parachain_id().into()).into();
```

Comment:
Do you query the parachain identifier here with `parachain_info`? What happens, when the parachain identifier changes (like it will happen for the main-net from `2101` to another id). Because the identifier is part of the Genesis configuration, we have to keep all usages of parachain_info in mind, right?

Reply:
Yes, it's take from `parachain_info`.
Our mainnet will not change the parachain id, it will always keep `2101` on Kusama. What will happen is that swap leases are swapped.
When the parachain id changes a lot of things happen, in that case the [location inversion](https://github.com/paritytech/polkadot/blob/3cf644abad63c4a177f0697683b72a64c4706852/xcm/xcm-builder/src/location_conversion.rs#L156-L204) could not be done properly anymore.